Makefile

```diff
@@ -25,6 +25,9 @@ ESP32_WIFI_SSID=
 ESP32_WIFI_PASSWORD=
 ESP32_PORT=
 
+# Use Toitware fork by default.
```

Comment:
Should we call this "Toitware's ESP-IDF fork" consistently?

Reply:
done.